File: api_sc_ate_condicionais/alunos/RIQUELME/api2023.2/milhas.js
import { question } from "readline-sync"

function milhas (){
    const nome = question('Digite o seu nome: ')

    const qtd_milhas = Number(question('Digite a quantidade de milhas (Só é possível comprar múltiplos de 1000): '))
    const milheiros = qtd_milhas / 1000
    const compra_milheiros = milheiros * 70


    const desconto = Number(question('  Digite o percentual de desconto: '))
    const desconto_total = calcular_desconto(compra_milheiros, desconto, desconto_total)
    const bonus = Number(question('Digite o percentual de bônus: '))


    console.log('*********** COMPRA DE MILHAS ****************')
    console.log(`Nome              : ${nome}`)
    console.log('Valor Ref.        : R$ 70,00/milheiro')
    console.log(`Milhas comprar    : ${qtd_milhas}`)
    console.log(`Desconto          : ${desconto_total}`)
    console.log(`Bônus             : ${bonus}`)




function calcular_desconto(preco_inicial, desconto, preco_final){
    const preco_inicial = Number(question('Preço inicial: '))
    const desconto = Number(question('Desconto: '))
    const preco_final = preco_inicial - (desconto/100)
    return preco_final


}








}
milhas()

File: api_sc_ate_condicionais/alunos/DavydLucas/milhas.py
def main():
    #Entrada
    milhas = int(input("Digite a quantidade de minlhas que deseja comprar(múltiplos de 1000): "))
    if milhas % 1000 == 0:
        desconto = int(input("Digite a quantidade do desconto:  (%)"))
            if desconto >= 0 and desconto <= 80:
                bonus = int("Digite o valor percentual do bônus entre 0 e 300(%): ")
                    if bonus >= 0 and bonus <= 300:
                                
                    else:
                        return 'Erro! Valor digitado fora da faixa.'            
            else:
                return 'Erro! Valor digitado fora da faixa.'
    else:
        return 'Não é possível fazer o cálculo com essas quantidades de milhas, apenas com múltiplos de 1000.'
    #Processamento

    #Saída


main()